src/components/PageHeader/AudioSetting.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Input, Text, useMatchBreakpoints } from '@pancakeswap-libs/uikit'
+import { useAudioModeManager } from 'state/user/hooks'
+import PancakeToggle from '../PancakeToggle/PancakeToggle'
+
+const StyledAudioSetting = styled.div`
+  margin-bottom: 16px;
+`
+
+const Label = styled.div`
+  align-items: center;
+  display: flex;
+  margin-bottom: 8px;
+`
+
+const Field = styled.div`
+  align-items: center;
+  display: inline-flex;
+
+  & > ${Input} {
+    max-width: 100px;
+  }
+
+  & > ${Text} {
+    font-size: 14px;
+    margin-left: 8px;
+  }
+`
+
+type AudioSettingModalProps = {
+  translateString: (translationId: number, fallback: string) => string
+}
+
+const AudioSetting = ({ translateString }: AudioSettingModalProps) => {
+  const { isSm, isXs } = useMatchBreakpoints()
+  const [audioPlay, toggleSetAudioMode] = useAudioModeManager()
+
+  return (
+    <StyledAudioSetting>
+      <Label>
+        <Text style={{ fontWeight: 600 }}>{translateString(999, 'Audio')}</Text>
+      </Label>
+      <Field>
+        <PancakeToggle scale={isSm || isXs ? 'sm' : 'md'} checked={audioPlay} onChange={toggleSetAudioMode} />
```

Comment:
I think you can use `scale={['sm,'md]}` (the styled system syntax) and avoid using the hook for styling

Reply:
That would be great yeah. Unfortunately its not possible here since its not styled-system component and UIKit doesn't have the helpers to implement this interface in custom components. `Toggle` would also require media query hook. It is definitely possible to integrate responsive array syntax into UIKit but I'd rather do it as a separate PR, its gonna take a while (see example [here](https://paulie.dev/posts/2020/08/styled-components-responsive-array-syntax/))

P.S. Hacky way would be to enclose it into Box container and scale it with array syntax but it doesn't work that well with some components, for example here pancakes will go ugly since they require some fine adjustments based on scale.